app/views/unattended/provisioning_templates/script/grubby_default.erb

```diff
@@ -3,7 +3,9 @@
 kind: script
 name: Grubby default
 model: ProvisioningTemplate
-%>
+description: |
+  TODO
```

Comment:
@lzap could you please provide a suggestion?

Reply:
Oh yeah, this is quite rare "script" template kind which is supposed to be just a template you can curl/wget to your server manually (or over ssh). https://projects.theforeman.org/issues/468

Surpringlisly, Grubby is still in EL8 for compatibility reasons: https://access.redhat.com/downloads/content/grubby/8.40-41.el8/x86_64/fd431d51/package

I think it makes sense to move this into ReX template, hey @jlsherrill interested in such patch? Do you still use this? :-)

Reply:
Description would be:

Script that can be manually downloaded to a server to reprovision it without booting from network.
Grubby compatibility package must be installed. WARNING: It will immediately reconfigure
bootloader and reprovision the node.
Usage:
  curl https://foreman/unattended/script -o reprovision.sh
  bash reprovision.sh
  reboot